hoth_api.yaml

```diff
@@ -456,11 +458,14 @@ components:
         missionOwner:
           type: string
         estimatedTaskOrderValue:
-          type: float
+          type: number
+          format: float
         initialAmount:
-          type: float
+          type: number
+          format: float
         remainingAmount:
-          type: float
+          type: number
+          format: float
         fundingIncrements:
```

Comment:
I'd mention this adjustment in the PR description if intentional

Reply:
I started to write something in the commit, but decided not. I'll add it in PR description.